packages/language-server/src/completion/completions.json

```diff
@@ -31,6 +31,17 @@
     {
       "label": "Decimal",
       "documentation": "Decimal value"
+    },
+    {
+      "label": "Unsupported(\"\")",
+      "documentation": "An arbitrary, unsupported database type",
```

Comment:
@do4gr Does this description make sense? This is for autocomplete

Reply:
@do4gr Maybe we could add the information that it is supported by Migrate and introspection but not Client?

Reply:
So it needs to be the valid raw sql needed to create the type in the database. We will interpolate it unchecked into the DDL. Migrate will run no validations on it. It will not be present in your client API since it is not supported there. 

Thats the necessary content, phrasing needs some work.

Reply:
I guess that goes into the tooltip below